.github/workflows/ci.yml

```diff
@@ -133,4 +149,4 @@ jobs:
       - name: Run tests
         run: yarn run start:test
         env:
-          MONGODB_URI: mongodb://localhost:27017/handshake
+          MONGODB_URI: mongodb://localhost:27017/handshake
```

Comment:
The database in the github workflow is an isolated docker container created just during the run of the script.
We want to use a temporary database, otherwise the tests would pollute the remote database everytime someone creates a pull request.